packages/minimongo/constants.js

```diff
@@ -0,0 +1,20 @@
+/** Exported values are also used in the mongo package. */
+
+/** @param {string} method */
+export function getAsyncMethodName(method) {
+  return `${method.replace('_', '')}Async`;
```

Comment:
Interesting -- would it make sense to keep the underscore prefix here so that the rule is simple? e.g., `_dropIndexAsync` to mirror `dropIndexAsync`? Otherwise it kind of turns a previously internal API into a public-looking async one. Maybe that is intentional though?

Reply:
It was like this when I overtook it and I think it's a good idea. These are around for a long time now and there's no clear alternative.